hw/ip/uart/data/uart.hjson

```diff
@@ -61,10 +61,14 @@
       type:    "status",
       default: "1",
       desc:    "raised if the transmit FIFO is past the high-water mark."}
+    { name:   "tx_empty",
```

Comment:
I understand wanting to keep the 'tx_' interrupts contiguous and separated from the 'rx_' ones although in this case I'd probably have been inclined to prioritize backwards compatibility and reduce the impact upon test software/SiVal needing to support multiple variants.
Perhaps agreement should be sought in this case? Renaming the existing interrupt helped with that, and it's unfortunate that adding another interrupt loses that advantage.